### DISCUSSION TITLE
External URL in OAuth2PasswordBearer

### DISCUSSION THREAD

**Discussion by adityashanbhog** [fastapi/fastapi#6933] [Questions] [answered]
I have created an api which authenticates the user using username and password, which is present in micro service A.
I want to use that authentication api in micro service B using
```Python
oauth2_scheme=OAuth2PasswordBearer(tokenUrl="/auth/token")
```
something like this. I want to put the authentication api URL which is present in a different micro service in tokenURL.
How should I achieve this as the api is present different micro service?

**Comment by Mause** [comment, marked as answer]
Please follow the issue template

**Comment by tiangolo** [comment]
Sorry, I (we) can't help you if you don't follow the issue template and provide more information.

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
